File: Assignment_4/Question1.c
#include<stdio.h>

int MultFact(int iNo)
{
    int iCnt = 0;
    int iMulti = 1;

    if(iNo < 0)
    {
        iNo = -iNo;
    }

    for(iCnt = 1; iCnt <=c iNo/2; iCnt++)
    {
        if(iNo % iCnt == 0)
        {
            iMulti = iMulti * iCnt;
        }
    }

    return iMulti;
}


int main()
{
    int iValue = 0;

    int iRet = 0;

    printf("Enter the number :\n ");
    scanf("%d", &iValue);

    iRet = MultFact(iValue);

    printf("The multiplication of the factors is : %d\n", iRet);

    return 0;
}

File: Assignment__18/Question1.java
// Write a program to check whether a number is prime or not.

import java.util.Scanner;

class logic
{
    boolean CheckPrime(int iNo)
    {
        int iCnt = 0;

        if(iNo < 0 )
        {
            iNo = -iNo;
        }

        if(iNo == 0 | iNo == 1)
        {
            System.out.println("Invalid Number.");
        }

        for(iCnt = 2; iCnt <= iNo/2; iCnt++)
        {
            if(iNo % iCnt == 0)
            {
                return false;
            }
        }
        return true;
    }
}

class Question1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        boolean bResult = false;

        logic lobj = new logic();

        bResult = lobj.CheckPrime(iValue);

        if(bResult == true)
        {
            System.out.println("The number is prime.");
        }
        else
        {
            System.out.println("The number is not prime.");  
        }

    }

}
